.github/CONTRIBUTING.md

```diff
@@ -50,16 +48,34 @@ Lifeycles include handling:
 - prerendering a compilation for production
 - fetching external (content) data sources
 
+## Project Management
+
+We take advantage of quite a few features on GitHub to assist in tracking issues, bugs, ideas and more for the project.  We feel that being organized not only helps the team in planning out priorities and ownership, it's also a great way to add visisbility and transparency to those following the project.
+
+### Project Boardss
+
+Our [sequentially named project boards](https://github.com/ProjectEvergreen/greenwood/projects) help us organize work in quartery buckets with a small handful of "top line" goals and objectives we would like to focus on for the upcoming time box.  It also serves as a catch-all for the usual work and bug fixes that happen throughout general maintenance of the project and can also yield good oppourtunities for those interested in contributing to see what we would appreciate help with the most.
+
+### Discussions
+
+We believe good collaboration starts with good communication.  As with most of the open source community, Greenwood is a 100% volunteer project and we understand the importance of respecting everyones [time and expectations](https://www.jason.af/setting-expectations).  Although we don't mind issues being made, unless the issue is clearly actionable and falls in-line with the motiviations and trajectory of the project, then feel free to go ahead an open a [Discussion](https://github.com/ProjectEvergreen/greenwood/discussions) first.
+
+We encourage discussions as we believe it is better to hash out technical discussions and proposals ahead of time since coding and reviewing PRs is very time consuming and as maintainer's, we want to make sure everyone gets the time they are deserved for contributing and this helps us plan our time in advance to best ensure a smooth flow of contributions through the project.
```

Comment:
"it is" can probably go with contraction instead.  Also this is a single sentence and it's long.